BigriverBookstore-apple/AppDelegate.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AppDelegate.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Cocoa
+import Spine
+
```

Comment:
@c-Viorel Just to make the review process easier and for us not focus on coding style, I propose to use something like https://github.com/nicklockwood/SwiftFormat. We can discuss on how to integrate it easier.